cedar-drt/fuzz/src/dump.rs

```diff
@@ -135,53 +132,62 @@ pub fn dump<'a>(
     Ok(())
 }
 
-/// Dump the entity uid to a string if it is specified, otherwise return `None`,
-/// which will appear as `null` in the JSON dump.
-fn dump_request_var(var: &EntityUIDEntry) -> Option<String> {
+/// Check whether a policy set can be successfully parsed
+fn well_formed(policies: &PolicySet) -> bool {
+    policies
+        .static_policies()
+        .map(ToString::to_string)
+        .all(|p| Policy::from_str(&p).is_ok())
+}
```

Comment:
is it sufficent to check the `.static_policies()`? should we also check templates?  is there an easy way to iterate over both static policies and templates?

Reply:
The current code only writes out the static policies because the relevant `abac*` targets only generate static policies. The general function is `.policies()`.

Reply:
Sounds good.